src/components/DocumentSelector/index.js

```diff
@@ -81,22 +83,20 @@ class DocumentSelector extends Component<Props & WithDefaultOptions> {
   }
 }
 
-const snakeToKebabCase = str => str.replace(/_/g, '-')
-
-const documentWithDefaultOptions = (types: Object, copyNamespace: 'identity' | 'proof_of_address') =>
+const documentWithDefaultOptions = (types: Object, group: groupType) =>
   (props: Props) =>
     <DocumentSelector
       {...props}
       defaultOptions={ i18n =>
         Object.keys(types).map(value => {
-          const { icon = `icon-${snakeToKebabCase(value)}`, hint, warning, ...other } = types[value]
+          const { icon = `icon-${kebabCase(value)}`, hint, warning, ...other } = types[value]
```

Comment:
Question: is this destructuring and assigning an object key at the same time?

Reply:
it's setting the default value if `type[value].icon` is `undefined`